lib/sdktest.pm

```diff
@@ -3,14 +3,25 @@ use base 'installedtest';
 use testapi;
 use utils;
 
+$sdktest::nightly_remote_name = 'eos-sdk';
+
 sub add_nightly_sdk_repo {
     my $self = shift;
     my $repo = 'http://endlessm.github.io/eos-knowledge-lib/eos-sdk-nightly.flatpakrepo';
+    my $collection_id = 'com.endlessm.Dev.Sdk';
 
     $self->disable_polkit_policy('org.freedesktop.Flatpak.configure-remote',
                                  '90-flatpak-configure-remote.rules');
     $self->user_console();
-    assert_script_run('flatpak remote-add --from eos-sdk-nightly ' . $repo);
+
+    # Do we actually need to add the remote? Nightly OS images have it enabled
+    # already. Note the inverted logic here because script_run() returns the
+    # exit status of its script.
+    if(script_run('flatpak remotes -d | grep -q ' . $collection_id) != 0) {
```

Comment:
Missing space before paren, but that's enough nitpicking for now. Feel free to fix that later.

Reply:
Gah, I don’t know my own coding style. I’ll push a fix for that.